recipes/r-boot/build.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
-
-# R refuses to build packages that mark themselves as Priority: Recommended
-mv DESCRIPTION DESCRIPTION.old
-grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
Thanks for the clarification. I  guess that I never found a package that needed that. (But I package only 2 r packages so far :grimacing: )